src/app/store/utils/vault-reducer-migration.ts

```diff
@@ -5,10 +5,14 @@ import type { initialKeysState } from '../keys/key.slice';
 
 const hiroWalletSalt = 'stacks-wallet-salt';
 const hiroWalletEncryptionKey = 'stacks-wallet-encrypted-key';
+export const oldWalletSalt = 'old-wallet-salt';
 
 export function migrateVaultReducerStoreToNewStateStructure(initialState: typeof initialKeysState) {
   const salt = localStorage.getItem(hiroWalletSalt);
   const encryptedSecretKey = localStorage.getItem(hiroWalletEncryptionKey);
+  // Save the previous wallet salt to later store on gaia encrypted
+  if (salt !== null) localStorage.setItem(oldWalletSalt, salt);
+
```

Comment:
This logic doesn't belong here. It has nothing to do with `migrateVaultReducerStoreToNewStateStructure`